LLReactiveMatchers/LLReactiveMatchersHelpers.m

```diff
@@ -24,6 +24,10 @@ extern BOOL LLRMCorrectClassesForActual(id object) {
     return [object isKindOfClass:RACSignal.class] || [object isKindOfClass:LLSignalTestRecorder.class];
 }
 
+extern BOOL LLRMCorrectClassesForExpected(id object, Class expectedClass) {
+  return [object isKindOfClass:expectedClass];
```

Comment:
I guess this method was added just in order to keep LLRM "conventions". I suggest adding 2 different methods: `LLRMCorrectClassesForError` and `LLRMCorrectClassesForValues`.